boards/arm/nucleo_g071rb/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_library()
```

Comment:
Need to protect under if(CONFIG_PINMUX).
Cf other boards following recent rebase

Reply:
done